uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/leave-request-ctrls.spec.js

```diff
@@ -1205,6 +1205,33 @@
       });
 
       describe('admin opens leave request popup in create mode', function () {
+        var adminId = 206;
+
+        beforeEach(function () {
+          $ctrl.request.contact_id = adminId.toString();
+
+          role = 'admin';
+          initTestController({
+            contactId: adminId
+          });
+        });
+
+        describe('on initialization', function () {
+          it('is in create mode', function () {
+            expect($ctrl.isMode('create')).toBeTruthy();
+          });
+
+          it('has admin role', function () {
+            expect($ctrl.isRole('admin')).toBeTruthy();
+          });
+
+          it('loads exactly one contact', function () {
+            expect(_.find($ctrl.managedContacts, { 'id': adminId })).toBeUndefined();
```

Comment:
if the statement is `it loads exactly one contact` shouldn't the test be something like `expect($ctrl.managedContacts.length).toBe(1)` or maybe the statement needs to change to something like `it does not include admin on contact list` ?